src/util/__tests__/to-markdown.ts

```diff
@@ -99,7 +101,54 @@ var a = 'b';
     expect(
       await toMarkdown(`
     Angular 1.x: \[addjs src="https://gist.github.com/JakePartusch/50737c37e988759e66b6.js?file=angular-controller.js"\] Angular 2.0: \[addjs src="https://gist.github.com/JakePartusch/d5863172113a92fc493d.js?file=angular2-component.ts"\]
-    `)
+    `).then(result => result.markdown)
     ).toMatchSnapshot();
   });
+
+  test('it translates old style gist blocks', async () => {
+    const html =
```

Comment:
Should probably be a fixture, but I 👍 this test!

Reply:
I'm not sure I know what a fixture is, but I agree :-)

Reply:
Create `src/util/__fixtures__/markdown-example.ts`

```ts
export const markdownExample = `
<!-- the content goes here -->
`;
```

then in the test

```ts
import { markdownExample } from '../__fixtures__/markdown-example.ts';

// profit
```

Reply:
Yep, I Googled it.  Already working on it